packages/suite-web/e2e/tests/coinmarket/buy.test.ts

```diff
@@ -0,0 +1,200 @@
+                    .should('be.equal', 'banxa');
+                cy.wrap(wrapper)
+                    .find('[class*="CoinmarketPaymentType__Wrapper"]')
+                    .invoke('text')
+                    .should('be.equal', 'Bank Transfer');
+            });
+
+        // Verifies receiving address and its title
+        cy.get('[class*="VerifyAddress__Wrapper"]')
+            .should('exist')
+            .then(wrapper => {
+                cy.wrap(wrapper)
+                    .find('[class*="AccountLabeling__TabularNums"]')
+                    .invoke('text')
+                    .should('be.equal', 'Bitcoin #1');
+            });
+
+        cy.get('[class*="AddressOptions__AddressWrapper"]')
+            .should('exist')
+            .then(wrapper => {
+                cy.wrap(wrapper)
+                    .find('[class*="AddressOptions__Address"]')
+                    .should('have.text', 'bc1q7ceqvaq7fqyywxqcx7qnfxkfk2ykpsla9pe80q');
+            });
+
+        // Moving on to the partner's site
+        cy.getTestElement('@coinmarket/buy/offers/finish-transaction-button').click();
+
+        // Verifies that the buy trade was approved
+        cy.get('[class*="PaymentSuccessful__Wrapper"]')
+            .should('exist')
+            .then(wrapper => {
+                cy.wrap(wrapper)
+                    .find('[class*="PaymentSuccessful__Title"]')
+                    .invoke('text')
+                    .should('be.equal', 'Approved');
+            });
+
+        // Goes back and verifies the transaction is listed under "Trade transactions"
+        cy.getTestElement('@coinmarket/back').click();
+        cy.get('[class*="BuyTransaction__Wrapper"]')
+            .first()
+            .should('exist')
+            .then(wrapper => {
+                cy.wrap(wrapper)
+                    .find('[class*="HiddenPlaceholder__Wrapper"]')
+                    .invoke('text')
+                    .should('match', new RegExp(`${testData.fiatInput} EUR`));
+                cy.wrap(wrapper)
+                    .find('[class*="FormattedCryptoAmount__Value"]')
+                    .invoke('text')
+                    .should('be.equal', testData.tradeBtcValue);
+                cy.wrap(wrapper)
+                    .find('[class*="FormattedCryptoAmount__Symbol"]')
+                    .should('contain.text', 'BTC');
+                cy.wrap(wrapper)
+                    .find('[class*="CoinmarketProviderInfo__Wrapper"]')
+                    .invoke('text')
+                    .should('be.equal', 'banxa');
+                cy.wrap(wrapper)
+                    .find('[class*="CoinmarketPaymentType__Text"]')
+                    .invoke('text')
+                    .should('be.equal', 'Bank Transfer');
+                cy.wrap(wrapper)
+                    .find('[class*="Status__Text"]')
+                    .invoke('text')
+                    .should('be.equal', 'Approved');
+            });
+    });
+});
```

Comment:
check other test files, you are going to need to add `export {}` to comply with current ts setup